js/iframeResizer.contentWindow.js

```diff
@@ -1101,4 +1103,4 @@
 
 	
 
-})(window || {});
+})(typeof window !== 'undefined' && window);
```

Comment:
no, it will throw an exception because the global variable `window` is not defined